sql/sql_table.cc

```diff
@@ -11624,7 +11624,14 @@ bool Sql_cmd_create_table_like::execute(THD *thd)
           (!thd->is_current_stmt_binlog_format_row() ||
            !create_info.tmp_table()))
       {
-        WSREP_TO_ISOLATION_BEGIN(create_table->db.str, create_table->table_name.str, NULL);
+#ifdef WITH_WSREP
+        WSREP_TO_ISOLATION_BEGIN_ALTER(create_table->db.str, create_table->table_name.str,
+				       first_table, &alter_info, NULL)
+	{
+	  WSREP_WARN("CREATE TABLE isolation failure");
```

Comment:
This warning comes up in tests galera.galera_split_brain, and galera.MDEV-22051 (see MTR run that was triggered for this pull request).
Either silence the warning in those two tests, or remove the warning from the code.